---
layout: default
title: Software
subtitle: Softwares in Liu Lab
permalink: /software/
---

# Softwares

### Cancer Immunotherapy
- [**TIMER**]() &nbsp;
"Tumor IMmune Estimation Resource, or TIMER, is a comprehensive resource for the clinical relevance of tumor-immune infiltrates, including B cells, CD4 and CD8 T cells, neutrophils, macrophages and dendritic cells."
- **TRUST** &nbsp;
"T cell receptor Repertoire Utilities for Solid Tissue/Tumor, or TRUST is a toolbox for analyzing T cell receptors in solid tumors using unselected RNA-seq data based on de novo assembly."
- **TIDE** &nbsp;
TIDE is a gene expression biomarker to predict the clinical response to immune checkpoint blockade. TIDE also provides gene-wise associations with T-cell dysfunction and immunotherapy resistance, computed from a large amount of cancer genomics datasets.

### CANCER PRECISION MEDICINE
- **CARE** &nbsp;
CARE is a software developed to enable large-scale inference of response biomarkers and drug combinations for targeted therapies using compound screen data.
- **TIDE** &nbsp;
TIDE is a gene expression biomarker to predict the clinical response to immune checkpoint blockade. TIDE also provides gene-wise associations with T-cell dysfunction and immunotherapy resistance, computed from a large amount of cancer genomics datasets.

### CRISPR ANALYSIS
- **MAGeCKFLute** &nbsp;
MAGeCKFlute is designed to support downstream analysis, utilizing the gene summary data provided through MAGeCK or MAGeCK-VISPR. Quality control, normalization, and screen hit identification for CRISPR screen data are performed in pipeline. Identified hits within the pipeline are categorized based on experimental design, and are subsequently interpreted by functional enrichment analysis.
- **MAGeCK-VISPR** &nbsp;
Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout + Visualization of CRISPR Screens (MAGeCK-VISPR) is a comprehensive pipeline for analyzing and visualizing results from genome-scale CRISPR-Cas9 knockout screens. 
- **MAGeCK** &nbsp;
Model-based Analysis of Genome-wide CRISPR-Cas9 Knockout (MAGeCK) is a computational tool to identify important genes from genome-scale CRISPR-Cas9 knockout screens.
- **NEST** &nbsp;
Network Essentiality Scoring Tool (NEST) is designed to predict gene essentiality based on protein interaction network and gene expression. It can also be used to enhance the quality of CRISPR screen results.  Publication submitted.
- **SSC** &nbsp;
Sequence Scan for CRISPR gRNA efficiency.

### CHIP-SEQ AND EPIGENETICS ANALYSIS
- **Cistrome-GO** &nbsp;
- **RABIT** &nbsp;
- **Cistrome** &nbsp;
- **Cistrome DB** &nbsp;
- **Model-based Analysis of ChIP-Seq (MACS)** &nbsp;
- **ChiLin** &nbsp;
- **BETA** &nbsp;
- **BINOCh** &nbsp;
- **Nucleosome Positioning from Sequencing (NPS)** &nbsp;
- **Cis-regulatory Element Annotation System(CEAS) - Standalone version** &nbsp;
- **Cis-regulatory Element Annotation System(CEAS)** &nbsp;
 
### TRANSCRIPTOME ANALYSIS
- **GFold** &nbsp;
- **SAGE-seq gene EXPRESSion analysis pipelines (sageExpress)** &nbsp;

### CHIP-CHIP ANALYSIS
- **Model-based Analysis of Tiling-array (MAT)** &nbsp;
- **Model-based Analysis of 2-Color Arrays (MA2C)** &nbsp;
- **extreme MApping of OligoNucleotides (xMAN)** &nbsp;
- **Hidden Markov model for analyzing ChIP-chip experiments on genome tiling arrays (HMMTiling)** &nbsp;
- **Microarray Blob Remover (MBR)** &nbsp;

### TRANSCRIPTION FACTOR MOTIF FINDING
- **CompareProspector** &nbsp;
- **Motif Regressor** &nbsp;
- **MDscan** &nbsp;
- **BioProspector** &nbsp;